ncm-interactivelimits/src/main/pan/components/interactivelimits/config.pan

```diff
@@ -5,5 +5,18 @@
 
 unique template components/${project.artifactId}/config;
 
-include { 'components/${project.artifactId}/config-common' };
-include { 'components/${project.artifactId}/config-rpm' };
+include { 'components/${project.artifactId}/schema' };
```

Comment:
remove the `{}`, they are not required to include pure strings, only when handling dml